File: tests/test_llm.py
import asyncio
from async_backend import backend


async def test():
    """Test."""
    print("Testing LLM...")
    async with backend:
        async for chunk in backend.send_message_async("Say only 'HELLO'"):
            # [X-Ray auto-fix] print(f"Chunk: {repr(chunk)}")
            pass


if __name__ == "__main__":
    asyncio.run(test())

File: rust_output/src/test_llm.rs
use anyhow::{Result, Context};
use crate::async_backend::{backen};
use tokio;

/// Test.
pub async fn test() -> () {
    // Test.
    println!("{}", "Testing LLM...".to_string());
    let _ctx = backend;
    {
        // async for
        while let Some(chunk) = backend.send_message_async("Say only 'HELLO'".to_string()).next().await {
            println!("Chunk: {}", repr(chunk));
            // pass
        }
    }
}
